.github/workflows/daily.yml

```diff
@@ -790,12 +790,15 @@ jobs:
         ./runtest-cluster ${{github.event.inputs.cluster_test_args}}
 
   test-macos-latest:
-    runs-on: macos-latest
+    runs-on: macos-13
```

Comment:
maybe we wanna keep running some basic sanity test on an older macos?
we do still have `macos-latest` in ci.yml, but that will some day shift to 13 too, so maybe we want it to be explicit (build on old and new variants).
and if we do that, and it's just for build, maybe all of that should all be in ci.yml, and we can leave daily.yml more straight forward?

Reply:
Seem that you are missing `will be removed` in the commit-msg.
It's just to verify that the fix is correct, i will revert it.